OmsAgent/apply_version.sh

```diff
@@ -15,9 +15,9 @@ sed -i "s/\"version\".*$/\"version\": \"$OMS_EXTENSION_VERSION\",/g" HandlerMani
 sed -i "s/^OMSExtensionVersion = .*$/OMSExtensionVersion = '$OMS_EXTENSION_VERSION'/"  watcherutil.py
 
 # updating omsagent.py
-# check BundleFileName = 'omsagent-1.12.7-0.universal.x64.sh'
+# check BundleFileName = 'omsagent-1.13.40-0.universal.x64.sh'
```

Comment:
Just make this generic; this comes from `omsagent.version`, don't want to have to update in multiple places each release